composer.json

```diff
@@ -27,7 +27,7 @@
     "autoload": {
         "psr-4": {
             "Zumba\\": "src/",
```

Comment:
@jrbasso If you specify `JsonSerializer` here, you can move the actual class to the root of the `src/` directory (per PSR4).